src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -344,7 +344,7 @@ class SharedLinkSection extends React.Component<Props> {
                 <hr />
                 {/* eslint-disable-next-line jsx-a11y/label-has-for */}
                 <label>
-                    <span className="label">
+                    <span className="label bdl-Label">
```

Comment:
why is `label` being kept here as well as adding `bdl-Label` ?

Reply:
I'd guess it's for backward compatibility, but since this isn't a `<Label>` component should this be something like `SharedLinkSection-label`? Or perhaps shouldn't be part of this change at all?

Reply:
It'd be on us to figure out why we have a regular label tag instead of the label component here. I can't recall, so might be some refactor we need to do.

Reply:
Yes, I'm keeping the old class names just in case there are overrides in EUA and AC. This `span` element is still using the `.label` class from Label.scss, so I think it's safe to add `.bdl-Label` to it.